kalite/caching/__init__.py

```diff
@@ -65,7 +68,11 @@ def invalidate_all_caches():
     call in here.
     """
     invalidate_inmemory_caches()
-    initialize_content_caches(force=True)
+    if settings.DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP:
```

Comment:
The underlying assumption here is that if generating in memory caches is too onerous a task to conduct at every system start up, then it is too onerous a task to conduct during server operation.

So we defer the regeneration of these caches to next system startup, by deleting the existing disk based copies of these caches.

This will prompt the caches to be recreated at next system start up, and the disk based copies to be rewritten.